### PR TITLE
Update extended-support-release.rst

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Comment by amyblais** [comment]
Add 
- Table to show timeline
- Any db changes

File: source/administration/extended-support-release.rst [hunk 1]
```diff
@@ -27,19 +31,34 @@ If your organization prefers to have the newest features and improvements, Exten
...
+Before you perform an upgrade, please ensure you have done a full back up of your current version.  You can restore the database and previous version if you need to revert an upgrade.  Please note that previous ESR versions are subject to an end of support date.
+
+Why is an Extended Support Release supported for 9 months and not longer?
+-------------------------------------------------------------------------
+We chose 9 months because the overhead for maintaining an ESR is high (and becomes higher the longer the support window is).
+If we increase the support window, it decreases how much we can develop the product. We specifically chose "ESR" instead of "LTS", since it is not intended to be a multi-year longterm support release. The 9 months support timeframe will not be extended in any case.
+
+Can customers pay for extended support?
+---------------------------------------
+At this point, we are not planning on letting customers pay for extended support, but we are open to discuss options for this.
```
Comment (wiersgallak) [inline source/administration/extended-support-release.rst]: 0/5 Maybe we indicate something along the lines of "Please speak to your Customer Success Manager if you have additional requirements for extended support" so they know their path of escalation?

Comment (amyblais) [reply source/administration/extended-support-release.rst]: Thanks! Fixed,

File: source/administration/release-lifecycle.rst [hunk 2]
```diff
@@ -1,67 +1,84 @@
 Release Lifecycle
 ========================
 
-What is an Release Lifecycle?
+What is a Release Lifecycle?
 ------------------------------------
-During each monthly release, Mattermost backports security and major bug fixes to the previous two monthly releases. Extended Support Releases receive backports for security fixes and major bug fixes for the length of their life cycle.  Once a release life cycle has ended, it will no longer receive the security or major bug backports. 
+During each monthly release, Mattermost backports security and major bug fixes to the previous two monthly releases. Extended Support Releases receive backports for security fixes and major bug fixes for the length of their life cycle.  Once a release life cycle has ended, it will no longer receive the security or major bug backports.
```
Comment (wiersgallak) [inline source/administration/release-lifecycle.rst]: > During each monthly release, Mattermost backports security and major bug fixes to the previous two monthly releases. 

Did this change to the previous 3 monthly releases recently?

Comment (amyblais) [reply source/administration/release-lifecycle.rst]: Yes, thank you, fixed.

File: source/administration/extended-support-release.rst [hunk 3]
```diff
@@ -27,19 +31,34 @@ If your organization prefers to have the newest features and improvements, Exten
...
 
 How do I restore a previous Extended Support Release?
 -----------------------------------------------------
-Before you perform an upgrade, please ensure you have done a full back up of your current version.  You can restore the database and previous version if you need to revert an upgrade.  Please note that previous ESR versions are subject to an end of support date. 
+Before you perform an upgrade, please ensure you have done a full back up of your current version.  You can restore the database and previous version if you need to revert an upgrade.  Please note that previous ESR versions are subject to an end of support date.
+
+Why is an Extended Support Release supported for 9 months and not longer?
+-------------------------------------------------------------------------
+We chose 9 months because the overhead for maintaining an ESR is high (and becomes higher the longer the support window is).
+If we increase the support window, it decreases how much we can develop the product. We specifically chose "ESR" instead of "LTS", since it is not intended to be a multi-year longterm support release. The 9 months support timeframe will not be extended in any case.
```
Comment (wiersgallak) [inline source/administration/extended-support-release.rst]: Add a space in "long[space]term.

File: source/administration/release-lifecycle.rst (hunk 2 above)
Comment (wiersgallak) [reply source/administration/release-lifecycle.rst]: I think this one still needs to be updated to three.  See it updated in the other doc.

**Pull request merged**